app/templates/components/orders/order-summary.hbs

```diff
@@ -88,7 +88,13 @@
                   {{ticket.name}}
                 </div>
               </td>
-              <td>{{currency-symbol this.eventCurrency}} {{format-money ticket.price}}</td>
+              <td>
+                {{#if (eq ticket.type 'free')}}
+                    {{t "Free"}}
+                  {{else}}
+                    {{currency-symbol this.eventCurrency}} {{format-money ticket.price}}
+                  {{/if}}
```

Comment:
Make this an HTML only component to avoid repetition. Just move it into its own HTML file and replace `ticket.type` and `ticket.price` to `@ticket.type` and vice versa. Then use that component and pass ticket like `@ticket={{ticket}}`

Reply:
Name the component `ticket-price.hbs`